app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/hplus/HPlusSupport.java

```diff
@@ -14,13 +14,16 @@
 import android.support.v4.content.LocalBroadcastManager;
 import android.widget.Toast;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
Sorry just saw Carsten already had that for Mi Band 2 ..
Ohhh and I wrote stupid code because I wanted to prevent another dependency back then...